example/lib/main.dart

```diff
@@ -7,7 +8,7 @@ void main() {
 }
 
 class Storyboard extends StatelessWidget {
-  const Storyboard({Key? key}) : super(key: key);
+  const Storyboard({Key key}) : super(key: key);
```

Comment:
Shouldn't we leave this as `Key?` ? I think we've used it that way everywhere else.